src/data/people.json

```diff
@@ -42,5 +42,27 @@
                 "link": "https://www.youtube.com/themsaid"
             }
         ]
+    },
+    {
+        "title": "Jeffrey Way",
+        "known_for": "Creator of Laracast",
```

Comment:
It should be `Laracasts`, not `Laracast`. :)

Reply:
fixed.